pkg/bisect/bisect.go

```diff
@@ -366,6 +371,13 @@ func (env *env) identifyRewrittenCommit() (string, error) {
 }
 
 func (env *env) minimizeConfig() (*testResult, error) {
+	var tasks []interface{}
+	if env.cfg.Kernel.DropSanitizers && env.crashTitle != "" {
```

Comment:
> It won't be feasible anyway until we move bug reproduction to syz-ci.

Yes, this may be a problem.

> And, even after we do it, we anyway have to deal with a lot of bugs that have a reproducer, but a non-minimized kernel config.

I don't think this is a serious problem. Cause bisections already happened for old bugs, and we try to obtain fresh reproducers for bug periodically. So old bugs will get minimized configs over time for fix bisections.

Users complained about too large configs for repros as well. Building these even on beefy machines is very slow.